it/src/test/java/com/google/cloud/teleport/it/jdbc/AbstractJDBCResourceManagerIT.java

```diff
@@ -78,20 +78,20 @@ public void testDefaultMSSQLResourceManagerE2E() {
   private <T extends AbstractJDBCResourceManager<?>> void simpleTest(T rm) {
     try {
       Map<String, String> columns = new LinkedHashMap<>();
+      columns.put("id", "INTEGER");
```

Comment:
nit This seems to be handled / added automatically based JDBCSchema.toSqlStatement

Reply:
I removed the method to add `id` column automatically. Instead, I made it a requirement to supply the id column in the map of fields supplied to JDBCSchema. 

I did this to allow non-integer primary key columns